### DISCUSSION TITLE
Table View for Attendances / Tabellenansicht für Anwesenheiten

### DISCUSSION THREAD

**Discussion by madetv1280** [lanis-mobile/lanis#276] [Ideas]
My idea is to display the attendances in a table, just like in the original, as this is somewhat clearer than the current card view that is being used in the app.

Also meine Idee ist wie beim Orginal die Anwsenheiten als Tabelle anzuzeigen da dies etwas Übersichtlicher ist also die akutell Kartenansicht wie sie in der App benutzt wird.

![test1](https://github.com/user-attachments/assets/b730f516-b865-43ef-a61c-89ac5e40015f)

**Comment by alessioc42** [comment]
Hi,
ich arbeite gerade an einem generellen Update für "Mein Unterricht". (#273)
Im Moment geht es darum, die eher redundanten Informationen umzustrukturieren. (z.B. nicht 3 Tabs und Hausaufgaben).
Ich werde mal sehen, wie ich den Wunsch umsetzen kann.

**Reply by alessioc42** [reply]
Leider ist es mir nicht gelungen, ein Design zu entwickeln, das auf dem Display eines Smartphones "anständig" funktioniert.
Mit dem nächsten Update wird es aber viel einfacher sein, unentschuldigte Fehlstunden bei Kursen zu erkennen. Die App zeigt diese dann direkt in der Kursmappenansicht hervorgehoben an.
